Packages/com.unity.inputsystem/InputSystem/InputSystem.cs

```diff
@@ -3553,6 +3553,10 @@ internal struct State
             [SerializeField] public string systemObject;
             #endif
             ////REVIEW: preserve InputUser state? (if even possible)
+            ///
```

Comment:
No triple slash comments except for doctools.

Looking at the REVIEW comment, probably also good to clarify for this one that it relates to domain reloads. Maybe should be a 

```
////TODO: make these saved states capable of surviving domain reloads
```

Reply:
I'll update the TODO/REVIEW and get rid of the slashes

Reply:
Fixed